### PR TITLE
REM-905: Add validation for send_raw_transaction method

### PR DESCRIPTION
**Pull request by DeV1doR**
Implements issue REM-905.

### CODE REVIEW

File: remme/tp/basic.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
 EMIT_EVENT = 'emit_event'
 PB_CLASS = 'pb_class'
 PROCESSOR = 'processor'
+VALIDATOR = 'validator'
```
Comment (eugene-babichenko) [inline remme/tp/basic.py]: Why no further use of that? Performing validation in TPs too at least will serve as a sanity check of those validators.

File: remme/rpc_api/transaction.py [hunk 2]
```diff
@@ -115,6 +155,24 @@
             message='Signer public key format is not valid'
         )
 
+    current_handler = None
+
+    for handler in TP_HANDLERS:
```
Comment (eugene-babichenko) [inline remme/rpc_api/transaction.py]: Why not just build a dict here?

File: remme/rpc_api/transaction.py [hunk 3]
```diff
@@ -115,6 +155,24 @@
...
+        if handler._family_name == tr_head_pb.family_name:
+            current_handler = handler
+            break
+
+    if current_handler:
```
Comment (eugene-babichenko) [inline remme/rpc_api/transaction.py]: There is no `else` branch which allows us not to perform any validation. I think it is better to throw an error here to force devs to write validators.

Comment (DeV1doR) [reply remme/rpc_api/transaction.py]: Agree

File: remme/rpc_api/transaction.py (hunk 2 above)
Comment (DeV1doR) [reply remme/rpc_api/transaction.py]: Not in this place, but could be

Comment (eugene-babichenko) [reply remme/rpc_api/transaction.py]: This might be better than iterating over a list :)

Comment (DeV1doR) [reply remme/rpc_api/transaction.py]: I mean to place dict in more global place)

**Pull request merged**